native_client/BUILD

```diff
@@ -55,47 +58,53 @@ tf_cc_shared_object(
            DECODER_SOURCES,
     # -Wno-sign-compare to silent a lot of warnings from tensorflow itself,
     # which makes it harder to see our own warnings
-    copts = ["-Wno-sign-compare", "-fvisibility=hidden"],
+    copts = ["-Wno-sign-compare", "-fvisibility=hidden"] + tflite_copts(),
```

Comment:
Can't combine `select({})` with `tflite_copts()` because it's already a `select()`, I'm open to a best solution :)

Reply:
This looks OK to me...